src/mpid/ch4/netmod/ofi/ofi_progress.c

```diff
@@ -15,23 +15,25 @@
 
 int MPIDI_OFI_progress(int vci, int blocking)
 {
-    int mpi_errno;
+    int mpi_errno, vni;
     struct fi_cq_tagged_entry wc[MPIDI_OFI_NUM_CQ_ENTRIES];
     ssize_t ret;
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_OFI_PROGRESS);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_OFI_PROGRESS);
 
+    vni = vci;
```

Comment:
This might benefit from a comment explaining how the vnis relate to the ch4 layer vci passed in this function.

Reply:
Yes.
```
For now, we have a one-to-one mapping between VCIs and VNIs
```

Reply:
Is there any near future plan to move away from the 1:1 mapping? 
If so, can a single vci_idx map to multiple vni_idx?

Reply:
Moving away from 1:1 mapping is not difficult but with performance concerns since this sits at key point of a fast path. We can easily add a hashing function or a netmod can replace with its own custom hashing function that manages the mapping in any scheme. We potentially will have to add multiple locks as the vci-layer locking (request pool) and lower-layer locking  may not align well.

Reply:
My original comment was suggesting a comment here in the code, explaining why are setting `vni = vci` and perhaps a TODO for later. I realize there is information in the commit message, but IMO a note here would improve things.

Reply:
I have updated the code to call `vni = MPIDI_OFI_vci_to_vni(vci)`, which is defined in `ofi_impl.h`. I could add TODO comments there.

Reply:
Function call is an improvement. TODO is optional for the folks who are hacking on this vni stuff :).